packages/block-library/src/image/index.php

```diff
@@ -0,0 +1,357 @@
+<?php
+/**
+ * core/image block.
+ */
+
+/**
+ * Warm the object cache with post and meta information for all found
+ * image blocks to avoid making individual database calls
+ * (similarly to `wp_make_content_images_responsive()`).
+ *
+ * @access private
+ * @since 4.2
```

Comment:
Should these be 5.0? 🔢

Reply:
Yeah, these need changing. Should be changed to "next Gutenberg" (4.3?) imho, then to 5.0 when merging to WP.